doc/cloud/cloud-cli.rst

```diff
@@ -0,0 +1,281 @@
+
+Devices
+-------
+
+Create Gateway
+''''''''''''''
+   .. code-block:: bash
+
+      knot-cloud create-gateway <name> <active>
+
+
+   .. code-block:: text
+
+      knot-cloud create-gateway gw1 false
+      {
+         "type": "knot:gateway",
+         "metadata": {
+            "name": "gw1"
+         },
+         "knot": {
+            "active": false,
+            "id": "1cb0b19a-f7bd-4a61-9f8d-f944746c039d"
+         },
+         "token": "ec587af2b5dd8f53adc0ddfb81cbfb38f1016685"
+      }
+
+Create Thing
+''''''''''''
+
+   .. code-block:: bash
+
+      knot-cloud create-thing <id> <name>
+
+
+   .. code-block:: text
+
+      knot-cloud create-thing f693054d669bf89c cli-thing
+
+         {
+            "type": "knot:thing",
+            "metadata": {
+               "name": "cli-thing"
+            },
+            "knot": {
+               "gateways": [],
+               "id": "f693054d669bf89c"
+            },
+            "token": "f2b374271be693a692261739437201be300bb7b4"
+         }
+
+Delete Device
+'''''''''''''
+
+   .. code-block:: bash
+
+      knot-cloud delete-device <id>
+
+   .. code-block:: text
+
+      knot-cloud delete-device f693054d669bf89c
+
+Create Session Token
+''''''''''''''''''''
+
+   .. code-block:: bash
+
+      knot-cloud create-session-token <id>
+
+   .. code-block:: text
+
+      knot-cloud create-session-token f693054d669bf89c
+      { token: 'f257798ac393d19b04dd29ac972b8417ade1f234' }
+
+List Devices
+''''''''''''
+
+   .. code-block:: bash
+
+      knot-cloud list-Devices
+
+   .. code-block:: text
+
+      knot-cloud list-Devices
+      [
+         {
+            "type": "knot:thing",
+            "metadata": {
+               "name": "cli-thing"
+            },
+            "knot": {
+               "gateways": [],
+               "id": "f693054d669bf89c"
+            }
+         },
+         {
+            "type": "knot:gateway",
+            "metadata": {
+               "name": "gw1"
+            },
+            "knot": {
+               "active": false,
+               "id": "1cb0b19a-f7bd-4a61-9f8d-f944746c039d"
+            }
+         }
+      ]
+
+Update Schema
+'''''''''''''
+
+You need to use the thing's credentials to update its credentials. Moreover, the CLI will update the schema with a default one if you need
+to update it quickly for testing purposes.
+
+   .. code-block:: bash
+
+      knot-cloud update-schema [sensor-id] [value-type] [unit] [type-id] [name] --credentials-file thing.json
+
+   .. code-block:: bash
+
+      knot-cloud update-schema 0 3 0 65521 Gw Thing
+
+Set Data
+''''''''
+
+Send a command to update a thing's sensor value.
+
+   .. code-block:: bash
+
+      knot-cloud set-data <thing-id> <sensor-id> <value>
+
+   .. code-block:: text
+
+      knot-cloud set-data f693054d669bf89c 1 true
+
+Get Data
+''''''''
+
+Send command to receive the last thing's sensor value.
```

Comment:
Add a comment saying that the command `knot-cloud listen-data` should be run in another terminal

Reply:
Done.